src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/ArchiveGoalTest.kt

```diff
@@ -178,7 +178,7 @@ class ArchiveGoalTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `should return 400 if other reason without description`() {
+  fun `should return 409 if other reason without description`() {
```

Comment:
I'm not sure about this .... 🤔 
To me if feels like it should be a 400 BAD REQUEST, in that the user has tried to archive a goal with the reason of OTHER, but has not provided a reason. From our swagger spec:
![Screenshot 2024-09-09 at 14 25 31](https://github.com/user-attachments/assets/663596bf-7039-4173-9a4d-f9776130ebf5)

I'm not sure this feels like a 409 ? , but if it were we would want to update the swagger spec to describe this behaviour.

Keep it as a 400? What do you think?

Reply:
400 is fine by me - we are using 409s for the status update changes so I aligned them but really I think you're correct in that 400 is more appropriate.